src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -110,9 +121,20 @@ func NewAdminAPI(urls []string, tlsConfig *tls.Config) (*AdminAPI, error) {
 		a.urls[i] = fmt.Sprintf("%s://%s", scheme, host)
 	}
 
+	if tryToMapBrokerIDstoHosts {
+		// TODO: Do not ignore the error.
+		// Intead, print a warning saying that RPK wasn't able to map brokers to hosts and
+		// it will broadcast write requests to all Admin brokers
+		a.brokerIdToUrls, _ = a.mapBrokerIDToHost()
```

Comment:
What's the reason to leave this out of this PR?

Reply:
Laziness 🙈
I'll add it to the PR